app/src/main/java/de/danoeh/antennapod/fragment/SearchFragment.java

```diff
@@ -241,14 +257,33 @@ public boolean onMenuItemActionCollapse(MenuItem item) {
 
     @Override
     public boolean onContextItemSelected(@NonNull MenuItem item) {
+
         FeedItem selectedItem = adapter.getLongPressedItem();
+        Feed selectedFeedItem  = adapterFeeds.getLongPressedItem();
+
+        int itemId = item.getItemId();
+        if ((itemId == R.id.remove_feed
+                || itemId == R.id.rename_folder_item
+                || itemId == R.id.remove_all_inbox_item
+                || itemId == R.id.edit_tags
+                || itemId == R.id.rename_item)
+                && selectedFeedItem != null) {
+            return FeedItemMenuHandler.onMenuItemClicked(this, item.getItemId(), selectedFeedItem);
+        }
```

Comment:
This would mean that all callers of `FeedItemMenuHandler.onMenuItemClicked` need to explicitly keep a list of all IDs. I was thinking about something like this:

```
if (FeedItemMenuHandler.onMenuItemClicked(this, item.getItemId(), selectedFeedItem)) {
    return true;
}
```